apps-rendering/src/components/series.tsx

```diff
@@ -56,85 +107,28 @@ const immersiveStyles = (
 	}
 `;
 
-const font = (isLabs: boolean): string =>
-	isLabs
-		? textSans.medium({ lineHeight: 'loose', fontWeight: 'bold' })
-		: headline.xxxsmall({ lineHeight: 'loose', fontWeight: 'bold' });
-
-const getLinkColour = (
-	{ liveblogKicker, kicker }: ThemeStyles,
-	isLabs: boolean,
-	isBlog: boolean,
-): string => {
-	if (isLabs) {
-		return palette.labs[300];
-	}
-	if (isBlog) {
-		return liveblogKicker;
-	}
-	return kicker;
-};
-
-const linkStyles = (
-	themeStyles: ThemeStyles,
-	isLabs: boolean,
-	isBlog: boolean,
-): SerializedStyles =>
-	css`
-		${font(isLabs)}
-		color: ${getLinkColour(themeStyles, isLabs, isBlog)};
-		text-decoration: none;
-
-		${darkModeCss`
-			color: ${isBlog ? themeStyles.liveblogKicker : themeStyles.inverted};
-		`}
-	`;
-
-const immersiveLinkStyles = (isLabs: boolean): SerializedStyles => css`
-	color: ${neutral[100]};
-	text-decoration: none;
-	white-space: nowrap;
-	${font(isLabs)}
+const standardStyles: SerializedStyles = css`
+	${articleWidthStyles}
+	padding-top: ${remSpace[1]};
 `;
 
-const getLinkStyles = (format: Format): SerializedStyles => {
-	const isLabs = isDisplayLabs(format);
-	const isBlog = isDesignBlog(format);
-
-	if (format.display === Display.Immersive) {
-		return immersiveLinkStyles(isLabs);
+const getStyles = ({ design, display, theme }: Format): SerializedStyles => {
+	if (display === Display.Immersive) {
+		return css(immersiveStyles(theme));
 	}
-	const themeStyles = getThemeStyles(format.theme);
-
-	return linkStyles(themeStyles, isLabs, isBlog);
-};
 
-const getStyles = (format: Format): SerializedStyles => {
-	const isLabs = isDisplayLabs(format);
-
-	if (format.display === Display.Immersive) {
-		return immersiveStyles(getThemeStyles(format.theme), isLabs);
-	}
-
-	if (
-		format.design === Design.LiveBlog ||
-		format.design === Design.DeadBlog
-	) {
-		return css(liveblogWidthStyles, liveblogPhabletSidePadding);
+	if (design === Design.LiveBlog || design === Design.DeadBlog) {
+		return css();
```

Comment:
Is this css empty because now that we are using container/column we don't need to have side paddings any more?

Reply:
Yes 👍